presto-orc/src/main/java/com/facebook/presto/orc/reader/LongDictionarySelectiveStreamReader.java

```diff
@@ -314,6 +317,7 @@ public void startStripe(Stripe stripe)
         readOffset = 0;
 
         presentStream = null;
+        rowGroupFilter = null;
```

Comment:
All readers: nullify rowGroupFilter in the `close()` as well?

Reply:
I blindly searched where presentStream is written and followed the pattern.